week-1-ts-ecosystem/w01e03-object-oriented-structure/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "target": "ES2022",
+    "lib": ["es6"],
```

Comment:
Dodanie tutaj tego `lib` sprawia że musimy podać pozostałe `lib`'s zawierające natywnie wykorzystane przez nas `interface` - ja bym to usunął bo dodstaje tego typu dziwaczne błędy:

![image](https://user-images.githubusercontent.com/25398949/194792330-c6944ae8-04cd-4b59-8614-9e1467a36047.png)